pipelines/rj_smtr/tasks.py

```diff
@@ -372,7 +372,7 @@ def query_logs(
 
 @task
 def get_raw(  # pylint: disable=R0912
-    url: str, headers: dict = None, filetype: str = "json", csv_args: dict = {}
+    url: str, headers: dict = None, filetype: str = "json", csv_args: dict = None
```

Comment:
@eng-rodrigocunha corrigi o parâmetro pois o dicionario vazio salva versoes anteriores. Melhor adotarmos o None como padrao e alterar para dict no body da funcao caso necessario

ref: https://stackoverflow.com/questions/26320899/why-is-the-empty-dictionary-a-dangerous-default-value-in-python

Reply:
Ótimo!